lib/jekyll/page.rb

```diff
@@ -87,17 +78,6 @@ def template
       end
     end
 
-    # The generated relative url of this page. e.g. /about.html.
-    #
-    # Returns the String url.
-    def url
-      @url ||= URL.new({
-        :template => template,
-        :placeholders => url_placeholders,
-        :permalink => permalink
-      }).to_s
-    end
-
```

Comment:
As far as I'm aware, Document (superclass) has the exact same `url` method (apart from renaming `template` to `url_template`), hence the removal here.